### ISSUE TITLE
shell_plus --print-sql Does Nothing

### ISSUE THREAD

**Issue by dsanders11** [django-extensions/django-extensions#1232]
As the title says, it does nothing.

The [implementation](https://github.com/django-extensions/django-extensions/blob/master/django_extensions/management/commands/shell_plus.py#L462) looks like it's hooking into a spot that's never going to be used, which is probably why it isn't working.

[This gist works](https://gist.github.com/aaugustin/2351479) but requires `DEBUG=True`, according to the [Django documentation](https://docs.djangoproject.com/en/2.1/topics/logging/#django-db-backends).

**Comment by devaljain1998** [comment]
I am using _django_extensions (version=2.2.5)_, which I guess is the latest version of the django_extensions but it is still not printing the SQL queries for some reason.
Any Help?

**Comment by trbs** [comment]
The "some reason" is that nobody has contributed a patch :-)
Readers should also take note of https://github.com/django-extensions/django-extensions/issues/1460 which is a different but similar issue.

Now what would help is if people would update this issue to be more clear or open a new one.

Working on #1460 I was also wondering about if `DEBUG=True` is required or not... I would prefer `--print-sql` to work regardlessly of what `DEBUG` is set too.

So in short; please update the title and description of this ticket or open a new one. Preferable with PR to have it function regardless of `settings.DEBUG`.
